journals/settings/utils.py

```diff
@@ -10,3 +16,18 @@ def get_env_setting(setting):
     except KeyError:
         error_msg = "Set the [%s] env variable!" % setting
         raise ImproperlyConfigured(error_msg)
+
+
+def get_whitelist_domains(request):
+    """
+        Extracts and returns domain list from siteconfiguration urls, including the domain on which server is running.
+    """
+    permitted_domains = ['', '{}:{}'.format(request.site.hostname, request.site.port)]
+    try:
+        for key, value in request.site.siteconfiguration.__dict__.items():
+            if key.endswith("url"):
```

Comment:
we should check if the fields contains `url` instead of endswith. since this fields like this can get skipped `ecommerce_journal_api_url`

Reply:
yeah I have update.